nncf/tensorflow/accuracy_aware_training/runner.py

```diff
@@ -17,14 +17,23 @@
 
 from nncf.common.utils.logger import logger as nncf_logger
 from nncf.common.accuracy_aware_training.runner import BaseAccuracyAwareTrainingRunner
+from nncf.common.accuracy_aware_training.runner import BaseAdaptiveCompressionLevelTrainingRunner
 from nncf.common.utils.helpers import configure_accuracy_aware_paths
 
 
-# pylint: disable=E1101
 class TFAccuracyAwareTrainingRunner(BaseAccuracyAwareTrainingRunner):
     """
     The Training Runner implementation for TensorFlow training code.
     """
+
+    # TODO(kshpv):: Should remove  *lr_updates_needed* param or keep it to save consistency with
+    #  PyTorch?
+    def __init__(self, training_params, verbose=True,
+                 validate_every_n_epochs=None, dump_checkpoints=True, lr_updates_needed=True):
```

Comment:
Could you give a short explanation of what the `lr_updates_needed` parameter means and why it needs in the Pytorch?

Reply:
Removed from TF part; 
In PyTorch this option defines whether NNCF should control lr_scheduler step call or not

Reply:
Created a ticket - 62723